src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/exceptions/PermanentStorageException.java

```diff
@@ -0,0 +1,10 @@
+package org.fogbowcloud.saps.engine.core.archiver.storage.exceptions;
+
+public class PermanentStorageException extends RuntimeException {
```

Comment:
again, anaemic abstraction. why do we need it? why not using RuntimeException directly?

Reply:
Added fixme.